comment_scan.rb

```diff
@@ -85,24 +85,23 @@ def restart(num_to_post, bundle)
             end
           when 'huh?'
             matched_regexes = report_raw(comment["post_type"], comment["body_markdown"])
-            reason_text = ""
-            #Go through regexes we matched to build reason_text
-            matched_regexes.each { |regex_matched| reason_text += "Matched reason \"#{Reason.where(id: regex_matched["reason_id"])[0]["name"]}\" for regex: #{regex_matched["regex"]}\n" }
-            #If post isn't deleted, check if this was an inactive comment
-            if !isPostDeleted(cli, comment["post_id"])
-              post = Array(cli.posts(comment["post_id"].to_i))[0]
-              if Time.at(post.json["last_activity_date"].to_i).to_date < Time.at(comment["creation_date"].to_i).to_date - 30
-                reason_text += "Comment was made #{(Time.at(comment["creation_date"].to_i).to_date - Time.at(post.json["last_activity_date"].to_i).to_date).to_i} days after last activity on post\n"
+            # Go through regexes we matched to build reason_text
+            reason_text = matched_regexes.map { |rm| "Matched reason \"#{Reason.where(id: rm["reason_id"]).first.name}\" for regex: #{rm.regex}" }.join("\n")
```

Comment:
Sorry, I think you took my comment a bit more literally than I'd intended :P Using `rm` for the variable name definitely worse than `regex_matched` -- long variable names are better because they're more informative. The more important part of the comment was that you should switch from the `{ .. }` 1-line syntax to the `do ... end` multiline syntax for better readability. Also, I'm pretty sure you can replace `Reason.where(id: rm["reason_id"]).first.name` with `rm.reason.name`.